### DISCUSSION TITLE
Problem with HAOS VM on proxmox

### DISCUSSION THREAD

**Discussion by SuneSondergaard** [tteck/Proxmox#2285] [Help Needed] [answered]
Hello

I'm having trouble trying to install "Home Assistant OS VM". It makes this error at the end of the script.

It ends up deleting "VM" after the error.

someone who can see what I'm doing wrong, or have overlooked. I am new to Proxmox

Regards Sune

![image](https://github.com/tteck/Proxmox/assets/46655654/d9165b9e-a325-4e33-8b8c-b759057dac54)

**Comment by tteck** [comment, marked as answer]
Did you make sure that **Virtualization is enabled**  in your machine’s BIOS.

**Reply by SuneSondergaard** [reply]
It had obviously been turned off again.

Thanks for the help, now it works.

**Comment by rishabdesai** [comment]
![image](https://github.com/tteck/Proxmox/assets/31618460/6ca89b51-426a-4758-a9b1-382a90cf7a95)

Getting error while installing Home Assistant on fresh installation of Proxmox 8.1.3.
steps taken: 
1) updated the proxmox package index 
2) Enable virtualization at bios level.
3) checked the proxmox is connected to internet using belo command:
```
if nc -zw1 8.8.8.8 443; then  echo "Internet Connected"; else  echo "Internet NOT Connected"; fi;
```
output is "Internet connected"

4) chekced if DNS is resolved using below command:
```
RESOLVEDIP=$(nslookup "github.com" | awk -F':' '/^Address: / { matched = 1 } matched { print $2}' | xargs)
if [[ -z "$RESOLVEDIP" ]]; then echo "DNS Lookup Failure";  else echo -e "DNS Resolved github.com to $RESOLVEDIP";  fi
```
output is : DNS Resolved github.com to 20.207.77.70

5) wget is already installed (version 1.21.3-1+b2)
(Ref: https://community.home-assistant.io/t/installing-home-assistant-os-using-proxmox-8/201835/781?u=rishabhdesai)

Thank you in advance.

**Reply by tteck** [reply]
Looking at your picture something is not right. Did you use the command from the website? Because default settings don't use that incomplete link.

**Reply by rishabdesai** [reply]
Thank you very much for your reply. 
After restarting the server and trying with link **bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/vm/haos-vm.sh)"
**, it worked perfectly. 

![image](https://github.com/tteck/Proxmox/assets/31618460/8f8e3b40-945e-40dc-a61c-5829afe834f8)
